app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from typing import (Dict, Any)
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)

```

Comment:
According to the checklist, it is recommended to use parentheses when importing multiple classes/modules. Please, consider changing this part of the code to follow this guideline. Here is an example of how it can be done:

python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError,
)
```